client/src/main/java/io/pravega/client/segment/impl/SegmentOutputStreamImpl.java

```diff
@@ -283,11 +284,13 @@ private Throwable getException() {
     private final class ResponseProcessor extends FailingReplyProcessor {
         @Override
         public void connectionDropped() {
+            log.info("Received connectionDropped for writer {}", writerId);
```

Comment:
Is this info or warn? In general, errors should be either `warn` or `error` level, depending on whether it is recoverable or not. In this case, it is not crystal clear because the error is dealt with automatically. I'd still prefer to make it warn, though. Same for the severity level for the other log messages changing in this file.

Reply:
Sure, this has changed now, as described in this comment: https://github.com/pravega/pravega/pull/3887#discussion_r295140458.